src/cbmc/cbmc_solvers.cpp

```diff
@@ -189,8 +187,6 @@ class cbmc_solver_with_filet:public cbmc_solverst::solvert
   ~cbmc_solver_with_filet()
   {
     // delete the prop before the file
-    delete prop_conv_ptr;
```

Comment:
The problem is that the deallocation order is reverse to the subclass relation and therefore overloading the destructor performs the deallocation in exactly the wrong order.

I'll discuss a solution for this with @mgudemann.